src/main/java/org/icgc_argo/workflow/search/controller/RunsApi.java

```diff
@@ -147,14 +147,15 @@
               value =
                   "OPTIONAL The preferred number of workflow runs to return in a page. If not provided, the implementation should use a default page size. The implementation must not return more items than `page_size`, but it may return fewer.  Clients should not assume that if fewer than `page_size` items are returned that all items have been returned.  The availability of additional pages is indicated by the value of `next_page_token` in the response.")
           @Valid
-          @RequestParam(value = "page_size", required = false)
-          Long pageSize,
+          @RequestParam(value = "page_size", defaultValue = "10", required = false)
+          Integer pageSize,
```

Comment:
I made this integer because the `size()` method in es search builder takes integer. Makes sense I guess cause es doesn't page more than 10,000 docs https://www.elastic.co/guide/en/elasticsearch/reference/current/paginate-search-results.html.